libpam/README.md

```diff
@@ -140,3 +143,11 @@ timebased, use the `google-authenticator` binary to generate a secret key in
 your home directory with the proper option.  In this mode, clock skew is
 irrelevant and the window size option now applies to how many codes beyond the
 current one that would be accepted, to reduce synchronization problems.
+
+## allow_perm
+
+By default, the PAM module requires the secrets file to be readable only by the owner of the file ( 0400 by default). In situations where the module is not used in a non-default configuration, an administrator may need more leanient file permissions, or a specific setting for their use case.
```

Comment:
Could you clarify what permissions would make sense in this case under an example use?

What does allow_perm set as allowed permissions?